src/api.js

```diff
@@ -1,5 +1,4 @@
 /* @flow */
-'use strict';
```

Comment:
Is this the right thing to do? Do you know why it is causing this lint error now?